wp-content/themes/archivepoliticalads/content-explore_sponsor_types.php

```diff
@@ -1,32 +1,50 @@
 <div class="row">
     <div class="col-md-12">
-        <div id="explore-sponsor_types-title" class="explore-subtitle-row">
-            <h3>Sponsor Types</h3>
-            <p>Browse ads by sponsor type</p>
+        <div id="explore-sponsor_types-title" class="explore-subtitle-row text-center">
+            <h3>Browse Ads by Sponsor Type</h3>
         </div>
     </div>
 </div>
 
 <div class="row">
-    <div class="col-md-12">
+    <div class="col-md-12 col-lg-8 col-lg-offset-2 ">
         <div id="explore-sponsor_types-content" class="explore-content-row">
-            <div id="explore-sponsor_types-slider" class="explore-list explore-slider">
-            <?php
-                $sponsor_types = get_sponsor_types();
-                foreach($sponsor_types as $sponsor_type) {
-                    ?>
-                    <a href="<?php bloginfo('url'); ?>/browse/?q=<?php echo(urlencode("sponsor_type:\"".$sponsor_type['name']."\""));?>">
-                        <div class="explore-item item">
-                            <div class="explore-wrapper">
-                                <div class="explore-label"><?php echo(get_sponsor_type_value($sponsor_type['name'])); ?></div>
-                                <div class="explore-count"><?php echo($sponsor_type['count']); ?> Ad<?php echo($sponsor_type['count']==1?"":"s");?></div>
+            <ol class="explore-list">
+                <?php
+                    $sponsor_types = get_sponsor_types();
+                    $first = true;
+                    $j = 0;
+                    foreach($sponsor_types as $sponsor_type) {
+
+                            if ($first) {
+                              $highestCount = $sponsor_type['count'];
```

Comment:
In the name of defensive programming I would "sort" the sponsor types first rather than relying on the return to always have highest count first.  (Same goes for the other places).  I could easily see that undocumented trait of the method changing since that isn't part of the formal method definition.